client/signup/steps/woocommerce-install/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useI18n } from '@wordpress/react-i18n';
+import { ReactElement } from 'react';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { GoToStep } from '../../types';
+import Complete from './complete';
+import Confirm from './confirm';
+import Install from './install';
+import Transfer from './transfer';
+
+import './style.scss';
+
+interface WooCommerceInstallProps {
+	goToStep: GoToStep;
+	stepName: string;
+	stepSectionName: string;
+	queryObject: {
+		siteSlug: string;
+	};
+}
+
+export default function WooCommerceInstall( props: WooCommerceInstallProps ): ReactElement | null {
+	const { __ } = useI18n();
+
+	return (
+		<StepWrapper
+			flowName="woocommerce-install"
+			hideSkip={ true }
+			nextLabelText={ __( 'Confirm' ) }
+			allowBackFirstStep={ true }
+			backUrl="/woocommerce-installation"
```

Comment:
Is this be `/start/woocommerce-install`? Or is this just another placeholder?

Reply:
Nope this is the url already exists and will be reworked (#57419) to be a landing page that gives people a way to enter the onboarding flow.